frank-flow/src/frontend/src/app/header/configuration-settings/configuration-settings.component.html

```diff
@@ -0,0 +1,105 @@
+<ngx-smart-modal
+  #configurationSettingsModal
+  class="configurationSettingsModal"
+  customClass="nsm-centered"
+  identifier="configurationSettingsModal"
+>
+  <div class="configurationSettings-container">
+    <h1>Configuration settings</h1>
+    <p>
+      If the value's are empty they can be added by selecting the desired option
+      in the dropdowns
```

Comment:
Say something about how the settings here are placed in the configuration, it assures that other people will view the configuration the same. If the user only wants to change it locally, the settings should be changed in the (global) settings. 

```suggestion
      These options change the way the Flow will apear for this particular configuration. The settings are saved in the configuration itself, so others will see the Flow the same way as you do. If you only want to change your view of the Flow, change the main <span (click)="closeFlowSettingsAndOpenSettingsModal()">Settings</span>
```

Adding some explanation can really improve the user experience.

Reply:
Added the text and function to code